examples/cluster/plot_digits_linkage.py

```diff
@@ -71,14 +75,12 @@ def plot_clustering(X_red, X, labels, title=None):
     plt.axis('off')
     plt.tight_layout()
 
-#----------------------------------------------------------------------
+# ----------------------------------------------------------------------
```

Comment:
I would just remove those separators, I found they just render the code unnecessarily heavier to read.